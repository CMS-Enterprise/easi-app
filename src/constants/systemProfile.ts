// Loosely grouped by System Subpages

// Team

export const teamSectionKeys = [
  'businessOwners',
  'projectLeads',
  'additional'
] as const;

export const TEAM_SECTION_MEMBER_COUNT_CAP = 5;
export const TOOLS_AND_SOFTWARE_PRODUCT_COUNT_CAP = 5;
export const BUDGET_ITEMS_COUNT_CAP = 5;

// ATO

export const ATO_STATUS_DUE_SOON_DAYS = 90;

// Threat levels match values from cedarThreat.weaknessRiskLevel
export const threatLevelGrades = [
  'Critical',
  'High',
  'Moderate',
  'Low',
  'Not Rated'
] as const;

export const securityFindingKeys = ['total', ...threatLevelGrades] as const;
