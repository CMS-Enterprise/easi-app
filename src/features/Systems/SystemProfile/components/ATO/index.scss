@use 'uswds-core' as *;
@use 'colors' as *;

#ato-and-security {
    .usa-card__container {
        border: 0;
        border-radius: 0;
        background-color: transparent;
    }

    .usa-process-list > .usa-process-list__item {
        max-width: none;
    }

    .border-success-darker {
        background-color: color(green-cool-60) !important;
    }
}
