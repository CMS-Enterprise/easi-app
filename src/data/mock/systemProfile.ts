import { cloneDeep } from 'lodash';

import GetSystemProfileQuery from 'queries/GetSystemProfileQuery';
import {
  GetSystemProfile,
  // eslint-disable-next-line camelcase
  GetSystemProfile_cedarSystemDetails_roles
} from 'queries/types/GetSystemProfile';
import { CedarAssigneeType } from 'types/graphql-global-types';
import { SystemProfileData } from 'types/systemProfile';
import { getSystemProfileData } from 'views/SystemProfile';

const assigneeUsername = 'JDOE';
const assigneeEmail = 'jane.doe@psu.edo';
const assigneeFirstName = 'Jane';
const assigneeLastName = 'Doe';

export const result: { data: GetSystemProfile } = {
  data: {
    cedarAuthorityToOperate: [
      {
        uuid: '00000000-0000-0000-0000-000000000000',
        tlcPhase: 'Operate',
        dateAuthorizationMemoExpires: '2022-07-30T00:00:00Z',
        countOfOpenPoams: 7,
        lastAssessmentDate: '2021-11-15T00:00:00Z',
        __typename: 'CedarAuthorityToOperate'
      }
    ],
    cedarThreat: [
      {
        weaknessRiskLevel: 'Moderate',
        __typename: 'CedarThreat'
      },
      {
        weaknessRiskLevel: 'Moderate',
        __typename: 'CedarThreat'
      },
      {
        weaknessRiskLevel: 'Moderate',
        __typename: 'CedarThreat'
      },
      {
        weaknessRiskLevel: 'Moderate',
        __typename: 'CedarThreat'
      },
      {
        weaknessRiskLevel: 'Moderate',
        __typename: 'CedarThreat'
      },
      {
        weaknessRiskLevel: 'Moderate',
        __typename: 'CedarThreat'
      },
      {
        weaknessRiskLevel: 'Low',
        __typename: 'CedarThreat'
      },
      {
        weaknessRiskLevel: 'High',
        __typename: 'CedarThreat'
      },
      {
        weaknessRiskLevel: 'Low',
        __typename: 'CedarThreat'
      },
      {
        weaknessRiskLevel: 'Low',
        __typename: 'CedarThreat'
      },
      {
        weaknessRiskLevel: 'Low',
        __typename: 'CedarThreat'
      },
      {
        weaknessRiskLevel: 'Low',
        __typename: 'CedarThreat'
      },
      {
        weaknessRiskLevel: 'Low',
        __typename: 'CedarThreat'
      },
      {
        weaknessRiskLevel: 'Low',
        __typename: 'CedarThreat'
      },
      {
        weaknessRiskLevel: 'Low',
        __typename: 'CedarThreat'
      },
      {
        weaknessRiskLevel: 'Moderate',
        __typename: 'CedarThreat'
      },
      {
        weaknessRiskLevel: 'Moderate',
        __typename: 'CedarThreat'
      },
      {
        weaknessRiskLevel: 'High',
        __typename: 'CedarThreat'
      },
      {
        weaknessRiskLevel: 'Moderate',
        __typename: 'CedarThreat'
      },
      {
        weaknessRiskLevel: 'Moderate',
        __typename: 'CedarThreat'
      },
      {
        weaknessRiskLevel: 'Moderate',
        __typename: 'CedarThreat'
      },
      {
        weaknessRiskLevel: 'Moderate',
        __typename: 'CedarThreat'
      },
      {
        weaknessRiskLevel: 'Moderate',
        __typename: 'CedarThreat'
      },
      {
        weaknessRiskLevel: 'Moderate',
        __typename: 'CedarThreat'
      },
      {
        weaknessRiskLevel: 'Low',
        __typename: 'CedarThreat'
      },
      {
        weaknessRiskLevel: 'Low',
        __typename: 'CedarThreat'
      },
      {
        weaknessRiskLevel: 'High',
        __typename: 'CedarThreat'
      },
      {
        weaknessRiskLevel: 'High',
        __typename: 'CedarThreat'
      },
      {
        weaknessRiskLevel: 'High',
        __typename: 'CedarThreat'
      },
      {
        weaknessRiskLevel: 'High',
        __typename: 'CedarThreat'
      },
      {
        weaknessRiskLevel: 'High',
        __typename: 'CedarThreat'
      },
      {
        weaknessRiskLevel: 'High',
        __typename: 'CedarThreat'
      },
      {
        weaknessRiskLevel: 'High',
        __typename: 'CedarThreat'
      },
      {
        weaknessRiskLevel: 'High',
        __typename: 'CedarThreat'
      },
      {
        weaknessRiskLevel: 'Moderate',
        __typename: 'CedarThreat'
      },
      {
        weaknessRiskLevel: 'Moderate',
        __typename: 'CedarThreat'
      },
      {
        weaknessRiskLevel: 'Moderate',
        __typename: 'CedarThreat'
      },
      {
        weaknessRiskLevel: 'Low',
        __typename: 'CedarThreat'
      },
      {
        weaknessRiskLevel: 'Low',
        __typename: 'CedarThreat'
      },
      {
        weaknessRiskLevel: 'Moderate',
        __typename: 'CedarThreat'
      },
      {
        weaknessRiskLevel: 'Moderate',
        __typename: 'CedarThreat'
      },
      {
        weaknessRiskLevel: 'Moderate',
        __typename: 'CedarThreat'
      },
      {
        weaknessRiskLevel: 'Low',
        __typename: 'CedarThreat'
      },
      {
        weaknessRiskLevel: 'Moderate',
        __typename: 'CedarThreat'
      },
      {
        weaknessRiskLevel: 'Low',
        __typename: 'CedarThreat'
      },
      {
        weaknessRiskLevel: 'Low',
        __typename: 'CedarThreat'
      },
      {
        weaknessRiskLevel: 'Low',
        __typename: 'CedarThreat'
      },
      {
        weaknessRiskLevel: 'Moderate',
        __typename: 'CedarThreat'
      },
      {
        weaknessRiskLevel: 'Low',
        __typename: 'CedarThreat'
      },
      {
        weaknessRiskLevel: 'High',
        __typename: 'CedarThreat'
      },
      {
        weaknessRiskLevel: 'High',
        __typename: 'CedarThreat'
      },
      {
        weaknessRiskLevel: 'High',
        __typename: 'CedarThreat'
      },
      {
        weaknessRiskLevel: 'High',
        __typename: 'CedarThreat'
      },
      {
        weaknessRiskLevel: 'High',
        __typename: 'CedarThreat'
      },
      {
        weaknessRiskLevel: 'High',
        __typename: 'CedarThreat'
      },
      {
        weaknessRiskLevel: 'High',
        __typename: 'CedarThreat'
      },
      {
        weaknessRiskLevel: 'High',
        __typename: 'CedarThreat'
      },
      {
        weaknessRiskLevel: 'Moderate',
        __typename: 'CedarThreat'
      },
      {
        weaknessRiskLevel: 'Moderate',
        __typename: 'CedarThreat'
      },
      {
        weaknessRiskLevel: 'Low',
        __typename: 'CedarThreat'
      },
      {
        weaknessRiskLevel: 'Moderate',
        __typename: 'CedarThreat'
      },
      {
        weaknessRiskLevel: 'Low',
        __typename: 'CedarThreat'
      },
      {
        weaknessRiskLevel: 'Moderate',
        __typename: 'CedarThreat'
      },
      {
        weaknessRiskLevel: 'Moderate',
        __typename: 'CedarThreat'
      },
      {
        weaknessRiskLevel: 'Low',
        __typename: 'CedarThreat'
      },
      {
        weaknessRiskLevel: 'Low',
        __typename: 'CedarThreat'
      }
    ],
    cedarSystemDetails: {
      businessOwnerInformation: {
        isCmsOwned: true,
        numberOfContractorFte: '20',
        numberOfFederalFte: '4',
        numberOfSupportedUsersPerMonth: '2750000',
        __typename: 'CedarBusinessOwnerInformation'
      },
      cedarSystem: {
        id: '000-100-0',
        name: 'CMS.gov',
        description:
          'CMS.gov provides the public and professionals the ability to access information regarding the CMS programs. CMS.gov website mission is to provide clear, accurate, and timely information about CMS programs to the entire health care community to improve quality and efficiency in an evolving health care system. CMS.gov website is a combination of static content and general content applications.  Core Function: * Information dissemination',
        acronym: 'cms.hhs.gov',
        status: 'Approved',
        businessOwnerOrg: 'Web and Emerging Technologies Group',
        businessOwnerOrgComp: 'OC',
        systemMaintainerOrg: 'Web and Emerging Technologies Group',
        systemMaintainerOrgComp: 'OC',
        __typename: 'CedarSystem'
      },
      deployments: [
        {
          id: '000-0000-1',
          dataCenter: {
            name: 'AWS US East/West',
            __typename: 'CedarDataCenter'
          },
          deploymentType: 'COOP DR',
          name: 'CMS.gov v.22 COOP DR',
          __typename: 'CedarDeployment'
        },
        {
          id: '000-0000-2',
          dataCenter: {
            name: 'AWS US East/West',
            __typename: 'CedarDataCenter'
          },
          deploymentType: 'Development',
          name: 'CMS.gov v.22 Development',
          __typename: 'CedarDeployment'
        },
        {
          id: '000-0000-3',
          dataCenter: {
            name: 'AWS US East/West',
            __typename: 'CedarDataCenter'
          },
          deploymentType: 'Production',
          name: 'CMS.gov v.22 Production',
          __typename: 'CedarDeployment'
        },
        {
          id: '000-0000-4',
          dataCenter: {
            name: 'AWS US East/West',
            __typename: 'CedarDataCenter'
          },
          deploymentType: 'Testing',
          name: 'CMS.gov v.22 Testing',
          __typename: 'CedarDeployment'
        },
        {
          id: '000-0000-5',
          dataCenter: {
            name: 'AWS US East/West',
            __typename: 'CedarDataCenter'
          },
          deploymentType: 'Implementation',
          name: 'CMS.gov v.22 Implementation',
          __typename: 'CedarDeployment'
        }
      ],
      roles: [
        {
          application: 'alfabet',
          objectID: '000-0000-0',
          roleTypeID: '238-17-0',
          assigneeType: CedarAssigneeType.ORGANIZATION,
          assigneeUsername: null,
          assigneeEmail: null,
          assigneeOrgID: '261-1497-0',
          assigneeOrgName: 'Web and Emerging Technologies Group',
          assigneeFirstName: null,
          assigneeLastName: null,
          roleTypeName: 'Business Owner',
          roleID: '384-16116-0',
          __typename: 'CedarRole'
        },
        {
          application: 'alfabet',
          objectID: '000-0000-0',
          roleTypeID: '238-28-0',
          assigneeType: CedarAssigneeType.ORGANIZATION,
          assigneeUsername: null,
          assigneeEmail: null,
          assigneeOrgID: '261-1497-0',
          assigneeOrgName: 'Web and Emerging Technologies Group',
          assigneeFirstName: null,
          assigneeLastName: null,
          roleTypeName: 'System Maintainer',
          roleID: '384-16406-0',
          __typename: 'CedarRole'
        },
        {
          application: 'alfabet',
          objectID: '000-0000-0',
          roleTypeID: '238-17-0',
          assigneeType: CedarAssigneeType.PERSON,
          assigneeUsername,
          assigneeEmail: null,
          assigneeOrgID: null,
          assigneeOrgName: null,
          assigneeFirstName,
          assigneeLastName,
          roleTypeName: 'Business Owner',
          roleID: '384-16670-0',
          __typename: 'CedarRole'
        },
        {
          application: 'alfabet',
          objectID: '000-0000-0',
          roleTypeID: '238-29-0',
          assigneeType: CedarAssigneeType.PERSON,
          assigneeUsername,
          assigneeEmail,
          assigneeOrgID: null,
          assigneeOrgName: null,
          assigneeFirstName,
          assigneeLastName,
          roleTypeName: 'ISSO',
          roleID: '384-17151-0',
          __typename: 'CedarRole'
        },
        {
          application: 'alfabet',
          objectID: '000-0000-0',
          roleTypeID: '238-32-0',
          assigneeType: CedarAssigneeType.PERSON,
          assigneeUsername,
          assigneeEmail: null,
          assigneeOrgID: null,
          assigneeOrgName: null,
          assigneeFirstName,
          assigneeLastName,
          roleTypeName: 'Project Lead',
          roleID: '384-17330-0',
          __typename: 'CedarRole'
        },
        {
          application: 'alfabet',
          objectID: '000-0000-0',
          roleTypeID: '238-53-0',
          assigneeType: CedarAssigneeType.PERSON,
          assigneeUsername,
          assigneeEmail: null,
          assigneeOrgID: null,
          assigneeOrgName: null,
          assigneeFirstName,
          assigneeLastName,
          roleTypeName: 'Survey Point of Contact',
          roleID: '384-17633-0',
          __typename: 'CedarRole'
        },
        {
          application: 'alfabet',
          objectID: '000-0000-0',
          roleTypeID: '238-51-0',
          assigneeType: CedarAssigneeType.PERSON,
          assigneeUsername,
          assigneeEmail: null,
          assigneeOrgID: null,
          assigneeOrgName: null,
          assigneeFirstName,
          assigneeLastName,
          roleTypeName: 'System Business Question Contact',
          roleID: '384-17820-0',
          __typename: 'CedarRole'
        },
        {
          application: 'alfabet',
          objectID: '000-0000-0',
          roleTypeID: '238-52-0',
          assigneeType: CedarAssigneeType.PERSON,
          assigneeUsername,
          assigneeEmail,
          assigneeOrgID: null,
          assigneeOrgName: null,
          assigneeFirstName,
          assigneeLastName,
          roleTypeName: 'System Data Center Contact',
          roleID: '384-18231-0',
          __typename: 'CedarRole'
        },
        {
          application: 'alfabet',
          objectID: '000-0000-0',
          roleTypeID: '238-50-0',
          assigneeType: CedarAssigneeType.PERSON,
          assigneeUsername,
          assigneeEmail,
          assigneeOrgID: null,
          assigneeOrgName: null,
          assigneeFirstName,
          assigneeLastName,
          roleTypeName: 'System Issues Contact',
          roleID: '384-18509-0',
          __typename: 'CedarRole'
        },
        {
          application: 'alfabet',
          objectID: '000-0000-0',
          roleTypeID: '238-50-0',
          assigneeType: CedarAssigneeType.PERSON,
          assigneeUsername,
          assigneeEmail,
          assigneeOrgID: null,
          assigneeOrgName: null,
          assigneeFirstName,
          assigneeLastName,
          roleTypeName: 'System Issues Contact',
          roleID: '384-18665-0',
          __typename: 'CedarRole'
        },
        {
          application: 'alfabet',
          objectID: '000-0000-0',
          roleTypeID: '238-28-0',
          assigneeType: CedarAssigneeType.PERSON,
          assigneeUsername,
          assigneeEmail,
          assigneeOrgID: null,
          assigneeOrgName: null,
          assigneeFirstName,
          assigneeLastName,
          roleTypeName: 'System Maintainer',
          roleID: '384-18958-0',
          __typename: 'CedarRole'
        },
        {
          application: 'alfabet',
          objectID: '000-0000-0',
          roleTypeID: '238-35-0',
          assigneeType: CedarAssigneeType.PERSON,
          assigneeUsername,
          assigneeEmail: null,
          assigneeOrgID: null,
          assigneeOrgName: null,
          assigneeFirstName,
          assigneeLastName,
          roleTypeName: "Contracting Officer's Representative (COR)",
          roleID: '384-19359-0',
          __typename: 'CedarRole'
        },
        {
          application: 'alfabet',
          objectID: '000-0000-0',
          roleTypeID: '238-55-0',
          assigneeType: CedarAssigneeType.PERSON,
          assigneeUsername,
          assigneeEmail,
          assigneeOrgID: null,
          assigneeOrgName: null,
          assigneeFirstName,
          assigneeLastName,
          roleTypeName: 'QA Reviewer',
          roleID: '384-19403-0',
          __typename: 'CedarRole'
        },
        {
          application: 'alfabet',
          objectID: '000-0000-0',
          roleTypeID: '238-56-0',
          assigneeType: CedarAssigneeType.PERSON,
          assigneeUsername,
          assigneeEmail,
          assigneeOrgID: null,
          assigneeOrgName: null,
          assigneeFirstName,
          assigneeLastName,
          roleTypeName: 'DA Reviewer',
          roleID: '384-20087-0',
          __typename: 'CedarRole'
        }
      ],
      urls: [
        {
          id: '000-0000-1',
          address: 'https://cms.gov',
          isAPIEndpoint: false,
          isBehindWebApplicationFirewall: true,
          isVersionCodeRepository: false,
          urlHostingEnv: 'Production',
          __typename: 'CedarURL'
        }
      ],
      systemMaintainerInformation: {
        agileUsed: true,
        deploymentFrequency: 'Bi-weekly',
        devCompletionPercent: '75% - 99%',
        devWorkDescription:
          'Continued enhancements and improvements and alignment with legislative changes.  Also, working on an upcoming redesign effort for the website.',
        netAccessibility:
          'Accessible to both public internet and to CMS-internal network',
        __typename: 'CedarSystemMaintainerInformation'
      },
      __typename: 'CedarSystemDetails'
    }
  }
};

export const query = {
  request: {
    query: GetSystemProfileQuery,
    variables: {
      cedarSystemId: '000-100-0'
    }
  },
  result
};

export function getMockSystemProfileData(
  data?: GetSystemProfile
): SystemProfileData {
  return getSystemProfileData(data ?? cloneDeep(result.data))!;
}

export function getMockPersonRole(
  // eslint-disable-next-line camelcase
  data?: Partial<GetSystemProfile_cedarSystemDetails_roles>
  // eslint-disable-next-line camelcase
): GetSystemProfile_cedarSystemDetails_roles {
  return {
    application: 'alfabet',
    objectID: '',
    roleTypeID: '',
    assigneeType: CedarAssigneeType.PERSON,
    assigneeUsername: null,
    assigneeEmail: null,
    assigneeOrgID: null,
    assigneeOrgName: null,
    assigneeFirstName: null,
    assigneeLastName: null,
    roleTypeName: null,
    roleID: null,
    __typename: 'CedarRole',
    ...data
  };
}
