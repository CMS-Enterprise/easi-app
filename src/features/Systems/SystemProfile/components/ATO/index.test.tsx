import React from 'react';
import { render } from '@testing-library/react';
import { getMockSystemProfileData } from 'tests/mock/systemProfile';

import ATO from './index';

const systemProfileData = getMockSystemProfileData();

describe('ATO subpage for System Profile', () => {
  it('matches snapshot', async () => {
    const { asFragment } = render(<ATO system={systemProfileData} />);
    expect(asFragment()).toMatchSnapshot();
  });

  it('renders a green card and expiration logic help text for Active', () => {
    const { getByTestId } = render(
      <ATO system={{ ...systemProfileData, atoStatus: 'Active' }} />
    );
    expect(getByTestId('Card')).toHaveClass('bg-success-dark');
    expect(getByTestId('atoExpirationLogicHelpText')).toBeInTheDocument();
  });

  it('renders a red card and expiration logic help text for Expired', () => {
    const { getByTestId } = render(
      <ATO system={{ ...systemProfileData, atoStatus: 'Expired' }} />
    );
    expect(getByTestId('Card')).toHaveClass('bg-error-dark');
    expect(getByTestId('atoExpirationLogicHelpText')).toBeInTheDocument();
  });

  it('renders a grey card and alert for No ATO', () => {
    const { getByTestId } = render(
      <ATO system={{ ...systemProfileData, atoStatus: 'No ATO' }} />
    );
    expect(getByTestId('Card')).toHaveClass('bg-base-lighter');
    expect(getByTestId('noATOAlert')).toBeInTheDocument();
  });
});
