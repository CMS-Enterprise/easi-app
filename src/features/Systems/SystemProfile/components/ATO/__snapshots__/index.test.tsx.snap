// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`ATO subpage for System Profile > matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="easi-section easi-section__border-bottom margin-bottom-4 padding-bottom-4"
    data-testid="section-wrapper"
  >
    <h2
      class="margin-top-0 margin-bottom-4"
      id="ato"
    >
      Authority to Operate (ATO)
    </h2>
    <ul
      class="usa-card-group margin-0"
      data-testid="CardGroup"
    >
      <li
        class="usa-card grid-col-12 bg-error-dark"
        data-testid="Card"
      >
        <div
          class="usa-card__container"
        >
          <header
            class="usa-card__header padding-2 padding-bottom-0 text-top text-white"
            data-testid="CardHeader"
          >
            <dt
              class="text-bold margin-bottom-05"
            >
              Status
            </dt>
            <dd
              class="description-definition margin-0 line-height-body-3 font-body-lg text-bold margin-bottom-2"
            >
              Expired
            </dd>
            <div
              class="easi-divider grid-col-12 border-error-darker"
            />
          </header>
          <div
            class="usa-card__footer padding-2 text-white"
            data-testid="CardFooter"
          >
            <div
              class="grid-row"
              data-testid="grid"
            >
              <div
                class="desktop:grid-col-6"
                data-testid="grid"
              >
                <dt
                  class="text-bold margin-bottom-05"
                >
                  Expiration Date
                </dt>
                <dd
                  class="description-definition margin-0 line-height-body-3 font-body-md"
                >
                  July 30, 2022
                </dd>
              </div>
              <div
                class="desktop:grid-col-6"
                data-testid="grid"
              >
                <dt
                  class="text-bold margin-bottom-05"
                >
                  Effective Date
                </dt>
                <dd
                  class="description-definition margin-0 line-height-body-3 font-body-md"
                >
                  February 26, 2022
                </dd>
              </div>
            </div>
          </div>
        </div>
      </li>
      <div
        class="easi-help-text display-flex"
        data-testid="atoExpirationLogicHelpText"
      >
        <svg
          class="usa-icon margin-right-1"
          focusable="false"
          height="1em"
          role="img"
          viewBox="0 0 24 24"
          width="1em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm1 15h-2v-6h2v6zm0-8h-2V7h2v2z"
          />
        </svg>
        Active ATOs will become Due Soon when they are 90 days away from their expiration
      </div>
    </ul>
    <div
      class="easi-section margin-bottom-4 margin-top-4"
      data-testid="section-wrapper"
    >
      <div
        class="usa-summary-box"
        data-testid="summary-box"
      >
        <div
          class="usa-summary-box__body"
        >
          <h3
            class="usa-summary-box__heading"
          >
            Security and privacy compliance at CMS
          </h3>
          <div
            class="usa-summary-box__text"
          >
            <p>
              Every information system operated by or on behalf of the U.S federal government is required to meet FISMA standards, which includes system authorization (ATO) signed by an Authoirizing Official (AO).
            </p>
            <div
              class="margin-top-1"
            >
              <a
                aria-label="Open 'Lean more about ATO' in a new tab"
                class="usa-link usa-link--external line-height-body-5"
                href="https://security.cms.gov/learn/authorization-operate-ato"
                target="_blank"
              >
                Learn more about ATO
                <span
                  aria-hidden="true"
                >
                   
                </span>
              </a>
            </div>
            <p>
              CMS FISMA Continuous Tracking System (CFACTS) is a CMS database that tracks application security deficiencies and POA&Ms, and supports the ATO process. IT is the CMS governance, risk, and compliance tool used as a repository to manage CMS' information systems security and privacy requirements.
            </p>
            <div
              class="margin-top-1 margin-bottom-2"
            >
              <a
                aria-label="Open 'CFACTS' in a new tab"
                class="usa-link usa-link--external line-height-body-5"
                href="https://cfacts.cms.gov"
                target="_blank"
              >
                Go to CFACTS
                <span
                  aria-hidden="true"
                >
                   
                </span>
              </a>
            </div>
            <div
              class="easi-divider"
            />
            <div
              class="margin-top-2"
            >
              <a
                aria-label="Open 'Cybergeek' in a new tab"
                class="usa-link usa-link--external line-height-body-5"
                href="https://security.cms.gov"
                target="_blank"
              >
                Learn more about security and privacy at CMS
                <span
                  aria-hidden="true"
                >
                   
                </span>
              </a>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    class="easi-section easi-section__border-bottom margin-bottom-4"
    data-testid="section-wrapper"
  >
    <h2
      class="margin-top-0"
      id="poamsAndFindings"
    >
      Plan of Action and Milestones (POA&Ms) and security findings
    </h2>
    <div>
      <div
        class="grid-row grid-gap margin-top-2"
        data-testid="grid"
      >
        <div
          class="tablet:grid-col-6 padding-right-2"
          data-testid="grid"
        >
          <dt
            class="text-bold margin-bottom-05"
          >
            Total Open POA&Ms
          </dt>
          <dd
            class="description-definition margin-0 line-height-body-3 margin-bottom-4"
          >
            7
          </dd>
        </div>
        <div
          class="tablet:grid-col-6 padding-right-2"
          data-testid="grid"
        >
          <dt
            class="text-bold margin-bottom-05"
          >
            Longest open POA&M
          </dt>
          <dd
            class="description-definition margin-0 line-height-body-3 margin-bottom-4"
          >
            255
          </dd>
        </div>
      </div>
    </div>
    <div
      class="grid-row grid-gap"
      data-testid="grid"
    >
      <div
        class="padding-right-2 grid-col-6"
        data-testid="grid"
      >
        <dt
          class="text-bold margin-bottom-05"
        >
          Total findings
        </dt>
        <dd
          class="description-definition margin-0 line-height-body-3 margin-bottom-4"
        >
          66
        </dd>
      </div>
      <div
        class="padding-right-2 grid-col-6"
        data-testid="grid"
      >
        <dt
          class="text-bold margin-bottom-05"
        >
          High findings
        </dt>
        <dd
          class="description-definition margin-0 line-height-body-3 margin-bottom-4"
        >
          18
        </dd>
      </div>
      <div
        class="padding-right-2 grid-col-6"
        data-testid="grid"
      >
        <dt
          class="text-bold margin-bottom-05"
        >
          Medium findings
        </dt>
        <dd
          class="description-definition margin-0 line-height-body-3 margin-bottom-4"
        >
          27
        </dd>
      </div>
      <div
        class="padding-right-2 grid-col-6"
        data-testid="grid"
      >
        <dt
          class="text-bold margin-bottom-05"
        >
          Low findings
        </dt>
        <dd
          class="description-definition margin-0 line-height-body-3 margin-bottom-4"
        >
          21
        </dd>
      </div>
    </div>
    <div
      class="grid-row grid-gap margin-top-2 margin-bottom-2"
      data-testid="grid"
    >
      <div
        class="tablet:grid-col-12"
        data-testid="grid"
      >
        <div
          class="usa-alert usa-alert--info"
          data-testid="alert"
        >
          <div
            class="usa-alert__body"
          >
            <p
              class="usa-alert__text"
            >
              If you have access to CFACTS, you can go there to view additional information about your system's POA&Ms and security findings. If you do not have access, ask this system's ISSO for help accessing addtional information in CFACTS.
            </p>
            <div
              class="margin-top-1"
            >
              <a
                aria-label="Open 'CFACTS' in a new tab"
                class="usa-link usa-link--external line-height-body-5"
                href="https://cfacts.cms.gov"
                target="_blank"
              >
                Go to CFACTS
                <span
                  aria-hidden="true"
                >
                   
                </span>
              </a>
            </div>
            <p />
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    class="easi-section"
    data-testid="section-wrapper"
  >
    <h2
      class="margin-top-0 margin-bottom-2"
      id="datesFormsAndTesting"
    >
      Dates, forms, and testing
    </h2>
    <div>
      <div
        class="grid-row grid-gap"
        data-testid="grid"
      >
        <div
          class="tablet:grid-col-6"
          data-testid="grid"
        >
          <dt
            class="text-bold margin-bottom-05"
          >
            Last ACT* or SCA*
          </dt>
          <dd
            class="description-definition margin-0 line-height-body-3 margin-bottom-4"
          >
            January 23, 2022
          </dd>
        </div>
        <div
          class="tablet:grid-col-6"
          data-testid="grid"
        >
          <dt
            class="text-bold margin-bottom-05"
          >
            Last penetration test
          </dt>
          <dd
            class="description-definition margin-0 line-height-body-3 margin-bottom-4"
          >
            May 31, 2022
          </dd>
        </div>
      </div>
      <div
        class="grid-row grid-gap"
        data-testid="grid"
      >
        <div
          class="tablet:grid-col-6"
          data-testid="grid"
        >
          <dt
            class="text-bold margin-bottom-05"
          >
            Last PIA*
          </dt>
          <dd
            class="description-definition margin-0 line-height-body-3 margin-bottom-4"
          >
            December 28, 2022
          </dd>
        </div>
        <div
          class="tablet:grid-col-6"
          data-testid="grid"
        >
          <dt
            class="text-bold margin-bottom-05"
          >
            Last ATO assessment
          </dt>
          <dd
            class="description-definition margin-0 line-height-body-3 margin-bottom-4"
          >
            November 15, 2021
          </dd>
        </div>
      </div>
      <div
        class="grid-row grid-gap"
        data-testid="grid"
      >
        <div
          class="tablet:grid-col-6"
          data-testid="grid"
        >
          <dt
            class="text-bold margin-bottom-05"
          >
            Last SIA*
          </dt>
          <dd
            class="description-definition margin-0 line-height-body-3 margin-bottom-4"
          >
            No data available
          </dd>
        </div>
        <div
          class="tablet:grid-col-6"
          data-testid="grid"
        >
          <dt
            class="text-bold margin-bottom-05"
          >
            Last Disaster Recovery exercise
          </dt>
          <dd
            class="description-definition margin-0 line-height-body-3 margin-bottom-4"
          >
            No data available
          </dd>
        </div>
      </div>
      <div
        class="grid-row grid-gap"
        data-testid="grid"
      >
        <div
          class="tablet:grid-col-6"
          data-testid="grid"
        >
          <dt
            class="text-bold margin-bottom-05"
          >
            Last Contengency Plan completion
          </dt>
          <dd
            class="description-definition margin-0 line-height-body-3 margin-bottom-4"
          >
            June 11, 2022
          </dd>
        </div>
        <div
          class="tablet:grid-col-6"
          data-testid="grid"
        >
          <dt
            class="text-bold margin-bottom-05"
          >
            Last Disaster Recovery Plan update
          </dt>
          <dd
            class="description-definition margin-0 line-height-body-3 margin-bottom-4"
          >
            No data available
          </dd>
        </div>
      </div>
    </div>
  </div>
  <div
    class="easi-section"
    data-testid="section-wrapper"
  >
    <div
      class="usa-table-container--scrollable"
      data-testid="scrollable-table-container"
      tabindex="0"
    >
      <table
        class="usa-table usa-table--borderless width-full"
        data-testid="table"
      >
        <thead>
          <tr>
            <th
              class="border-bottom-2px"
              scope="col"
            >
              Document
            </th>
            <th
              class="border-bottom-2px"
              scope="col"
            >
              Does this sytem have one?
            </th>
          </tr>
        </thead>
        <tbody>
          <tr>
            <td>
              Disaster Recovery Plan
            </td>
            <td>
              No data available
            </td>
          </tr>
          <tr>
            <td>
              SORN*
            </td>
            <td>
              Yes
            </td>
          </tr>
          <tr>
            <td>
              Contingency Plan
            </td>
            <td>
              Yes
            </td>
          </tr>
        </tbody>
      </table>
    </div>
  </div>
  <div
    class="easi-section margin-bottom-4 margin-top-4"
    data-testid="section-wrapper"
  >
    <div
      class="margin-top-3 padding-2 bg-base-lightest"
    >
      <h4
        class="margin-top-0"
      >
        *Acronyms defined
      </h4>
      <a
        aria-label="Open 'Full list of security acronyms' in a new tab'"
        class="usa-link usa-link--external line-height-body-5"
        href="https://security.cms.gov/learn/acronyms"
        target="_blank"
      >
        Full list of security related acronyms
        <span
          aria-hidden="true"
        >
           
        </span>
      </a>
      <p>
        ACT: Adaptive Capabilities Testing
      </p>
      <p>
        SCA: Security Control Assessment
      </p>
      <p>
        PIA: Privacy Impact Assessment
      </p>
      <a
        aria-label="Open 'Lean more about PIA' in a new tab"
        class="usa-link usa-link--external line-height-body-5"
        href="https://security.cms.gov/learn/privacy-impact-assessment-pia"
        target="_blank"
      >
        Learn more about PIA
        <span
          aria-hidden="true"
        >
           
        </span>
      </a>
      <p>
        SIA: Security Impact Analysis
      </p>
      <a
        aria-label="Open 'Lean more about SIA' in a new tab"
        class="usa-link usa-link--external line-height-body-5"
        href="https://security.cms.gov/learn/security-impact-analysis-sia"
        target="_blank"
      >
        Learn more about SIA
        <span
          aria-hidden="true"
        >
           
        </span>
      </a>
      <p>
        SORN: System of Records Notice
      </p>
      <a
        aria-label="Open 'Lean more about SORN' in a new tab"
        class="usa-link usa-link--external line-height-body-5"
        href="https://security.cms.gov/learn/system-records-notice-sorn"
        target="_blank"
      >
        Learn more about SORN
        <span
          aria-hidden="true"
        >
           
        </span>
      </a>
    </div>
  </div>
</DocumentFragment>
`;
